test/e2e.ens.js

```diff
@@ -50,5 +57,126 @@ describe('ENS [ @E2E ]', function () {
             '0x0000000000000000000000000000000000000001'
         );
     });
+
+    it('should get/set a publickey', async function(){
```

Comment:
Some great new test cases thanks Chris! But also here.. aren't there unnecessary changes that would be against the guidelines of this project? :-)

Reply:
Idk. This PR introduces to interface validation as part of the solution to supporting contentHash per design discussion in #3392. We have existing methods that will be validated this way so we should also test them.